load-testing/src/main/java/google/registry/client/EppClient.java

```diff
@@ -365,17 +383,55 @@ public void run() {
         }
       }
 
+      for (ChannelFuture channelFuture : channelFutures) {
+        Channel channel = channelFuture.channel();
+        int channelNumber = channel.attr(CHANNEL_NUMBER).get();
+        int responsesReceived = channel.attr(RESPONSE_RECEIVED).get().size();
+        for (int i = 0; i < responsesReceived; i++) {
+          requestDurations.add(
+              Duration.between(
+                      channel.attr(REQUEST_SENT).get().get(i),
+                      channel.attr(RESPONSE_RECEIVED).get().get(i))
+                  .toMillis());
+        }
+        if (responsesReceived != requestsPerConnection) {
+          incompleteConnections.add(channelNumber);
+          failedRequests += (requestsPerConnection - channel.attr(REQUEST_SENT).get().size());
+        }
+        if (responsesReceived > 0) {
+          startTime = updateStartTime(channelFutures, channelNumber, startTime);
+          endTime = updateEndTime(channelFutures, channelNumber, endTime);
+        }
+      }
+
+      if (!incompleteConnections.isEmpty()) {
+        System.out.printf("%d incomplete connections: ", incompleteConnections.size());
+        for (int channelNumber : incompleteConnections) {
+          System.out.printf("Channel %d not finished\n", channelNumber);
+        }
+        System.out.print("\n");
+      }
+
       System.out.println();
       System.out.println("====== SUMMARY ======");
       System.out.printf("Number of connections: %d\n", connections);
-      System.out.printf("Number of requests per connection: %d\n", requestPerConnection);
+      System.out.printf("Number of requests per connection: %d\n", requestsPerConnection);
+      if (startTime != null && endTime != null) {
+        System.out.printf(
+            "QPS: %.2f\n",
```

Comment:
Add a better label here that clarifies exactly what this is.

Is it request QPS?  Response QPS?  Domains created successfully QPS?